### ISSUE TITLE
Use of \concept or \keyword in documentation 

### ISSUE THREAD

**Issue by kylehamilton** [wviechtb/metadat#7]
Currently, in the documentation we are using \concept to describe what the datasets look like. A lot of the tags are for the academic fields of publication such as education or medicine. Do we want this to be how we organize the types of datasets or do we want to use \keyword to describe these datasets? The Writing R Extensions manual says "each \concept entry should give a single index term", for now, I went back into the documentation and removed any case where I used two words in a \concept field but there are a few others currently there. I just wanted to wait until I got direction to make sure I didn't break something.

https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Indices

**Comment by wviechtb** [comment]
The \keyword entries are somewhat standardized (see file KEYWORDS in R_HOME/doc). Of course one can add additional ones, but we decided not to do that. See also: https://cran.r-project.org/doc/manuals/r-release/R-exts.html#index-_005ckeyword where it says "Do strongly consider using \concept (see Indices) instead of \keyword if you are about to use more than very few non-standard keywords." So we went with \concept. And yes, each \concept entry should give a single index term (but there can be multiple entries). We still need to start cataloguing all the terms we use (so that people don't use different terms for the same concept).
